clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -304,6 +304,7 @@ public void releaseServerStoreProxy(ServerStoreProxy storeProxy, boolean isRecon
     if (!isReconnect) {
       storeProxy.close();
     } else {
+      LOGGER.info("Adding to reconnect set");
```

Comment:
What would this log mean to a user?

Reply:
This needs to go, I added for debugging and forgot